packages/gasket-plugin-elastic-apm/lib/index.js

```diff
@@ -46,41 +45,36 @@ module.exports = {
         );
 
         if (!apm.isStarted()) {
-          logger.notice('WARNING Elastic APM agent is not started. Use `--require elastic-apm-node/start`');
+          logger.notice('WARNING Elastic APM agent is not started. Use `--require ./setup.js`');
         }
 
-        apm.addFilter(filterSensitiveCookies(config));
+        if (config.elasticAPM && config.elasticAPM.sensitiveCookies) {
+          logger.notice('WARNING: elasticAPM.sensitiveCookies has been removed. Filter sensitive data in the setup.js script.');
+        }
       }
     },
     create: {
       timing: {
         after: ['@gasket/plugin-start']
       },
-      handler(gasket, { pkg }) {
+      handler(gasket, { pkg, files }) {
+        const generatorDir = `${ __dirname }/../generator`;
+
         pkg.add('dependencies', {
-          'elastic-apm-node': dependencies['elastic-apm-node']
+          'elastic-apm-node': dependencies['elastic-apm-node'],
+          'dotenv': dependencies.dotenv
```

Comment:
Per my other comment, let's have these be `devDependencies` to allow apps to control the version better - especially since they are added to the package during create anyway.
```suggestion
          'elastic-apm-node': devDependencies['elastic-apm-node'],
          'dotenv': devDependencies.dotenv
```